grpc/src/main/java/io/stargate/grpc/service/Service.java

```diff
@@ -0,0 +1,218 @@
+package io.stargate.grpc.service;
+
+import com.github.benmanes.caffeine.cache.Cache;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import io.grpc.Context;
+import io.grpc.Status;
+import io.grpc.StatusException;
+import io.grpc.StatusRuntimeException;
+import io.grpc.stub.StreamObserver;
+import io.stargate.auth.AuthenticationSubject;
+import io.stargate.core.metrics.api.Metrics;
+import io.stargate.db.AuthenticatedUser;
+import io.stargate.db.ClientInfo;
+import io.stargate.db.ImmutableParameters;
```

Comment:
Intellij complains this is a missing class, but could be something local to me

Reply:
It means that the generated code for `ImmutableParameters` wasn't properly attached to the project.